web/client/components/map/cesium/Map.jsx

```diff
@@ -335,6 +340,12 @@ class CesiumMap extends React.Component {
         this.pauserStream$ = pauserStream$;
     };
     registerHooks = () => {
+        // Unregister hooks as coming from a leaflet or openlayer map retains hooks
+        // causing issue in feature info click
+        this.props.hookRegister.registerHook(GET_PIXEL_FROM_COORDINATES_HOOK);
```

Comment:
in the comment you say unregister, but then you use **registerHook**

can you clarify?

Reply:
Yes. Here I'm registering the hook GET_PIXEL_FROM_COORDINATES_HOOK with an undefined value which should return undefined when performing a getHook('GET_PIXEL_FROM_COORDINATES_HOOK'). So in a sense we are unregistering the hook.